prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+    var $container = document.querySelector('div.container');
+    console.log(container);
+    var $heading = document.createElement('h1');
+    $heading.textContent = 'Hello World';
+    console.log($heading);
+    $heading.appendChild($container);
```

Comment:
Here you want to append `$heading` into `$container` so it should be the other way around compared to your code. `$container.appendChild($heading);`.